### PR TITLE
Add linux.cpu.clock and os.cpu.clock

### PR DESCRIPTION
**Pull request by gbrayut**

### CODE REVIEW

File: cmd/scollector/collectors/collectors.go [hunk 1]
```diff
@@ -43,6 +44,7 @@ const (
 )
 
 const (
+	osCPUClockDesc     = "Current speed of the processor in MHz."
```
Comment (kylebrandt) [inline cmd/scollector/collectors/collectors.go]: The current

File: cmd/scollector/collectors/procstats_linux.go [hunk 2]
```diff
@@ -169,6 +170,20 @@ func c_procstats_linux() (opentsdb.MultiDataPoint, error) {
...
+	cpuinfo_index := 0
+	if err := readLine("/proc/cpuinfo", func(s string) error {
+		m := cpuspeedRE.FindStringSubmatch(s)
+		if m == nil {
+			return nil
+		}
+		tags := opentsdb.TagSet{"cpu": strconv.Itoa(cpuinfo_index)}
+		Add(&md, osCPUClock, m[1], tags, metadata.Gauge, metadata.MHz, osCPUClockDesc)
+		Add(&md, "linux.cpu.clock", m[1], tags, metadata.Gauge, metadata.MHz, osCPUClockDesc)
+		cpuinfo_index += 1
```
Comment (mjibson) [inline cmd/scollector/collectors/procstats_linux.go]: What is the purpose of this line? cpuinfo_index is never used after this.

File: cmd/scollector/collectors/procstats_linux.go [hunk 3]
```diff
@@ -169,6 +170,20 @@ func c_procstats_linux() (opentsdb.MultiDataPoint, error) {
 	if num_cores != 0 && t_util != 0 {
 		Add(&md, osCPU, t_util/float64(num_cores), nil, metadata.Counter, metadata.Pct, "")
 	}
+	cpuinfo_index := 0
```
Comment (mjibson) [inline cmd/scollector/collectors/procstats_linux.go]: Why is this here? It's always 0 when it is used.

**Comment by mjibson** [comment]
Disregard my comments if you got an email about them. Didn't see that it was in a readLine and was thus in a loop.
